AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -55,18 +115,54 @@ internal final class QRCodeComponent: ActionComponent, Localizable {
         return QRCodeView.Model(
             instruction: ADYLocalizedString("adyen.pix.instructions", localizationParameters),
             logoUrl: url,
+            observedProgress: progress,
+            expiration: $expirationText,
             style: QRCodeView.Model.Style(
                 copyButton: style.copyButton,
                 instructionLabel: style.instructionLabel,
+                progressView: style.progressView,
+                expirationLabel: style.expirationLabel,
                 backgroundColor: style.backgroundColor
             )
         )
     }
+    
+    private lazy var awaitComponent: AnyAwaitActionHandler = {
+        let client = APIClient(environment: environment)
+        let scheduler = BackoffScheduler(queue: .main)
+        let retry = RetryAPIClient(apiClient: client, scheduler: scheduler)
```

Comment:
the `retry` `scheduler ` and `client ` are not used, right?

Reply:
👍